apply/golden_file_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !online
```

Comment:
shouldn't it be `offline`?

Reply:
I just pushed a diff to invert this. Now it is `!offline` meaning "build this if we haven't set the offline tag".